doc/SAI-Proposal-ACL-1.md

```diff
@@ -0,0 +1,385 @@
+    acl_entry_attrs[1].id = SAI_ACL_ENTRY_ATTR_PRIORITY;
+    acl_entry_attrs[1].value.u32 = 1;
+    acl_entry_attrs[2].id = SAI_ACL_ENTRY_ATTR_FIELD_SRC_IP;
+    CONVERT_STR_TO_IP(acl_entry_attrs[2].value.aclfield.data.ip4, "192.168.100.100");
+    acl_entry_attrs[3].id = SAI_ACL_ENTRY_ATTR_FIELD_L4_SRC_PORT; 
+    acl_entry_attrs[3].value.aclfield.data.u16 = 1000;
+    saistatus = sai_acl_api->create_acl_entry(&acl_entry, 4, acl_entry_attrs);
+    if (saistatus != SAI_STATUS_SUCCESS) {
+        return saistatus;
+    }
+    
+    // Bind this ACL table to a router interface *rifid10* 
+    rif_attr_list.count = 1;
+    rif_attr_list.list[0].id = SAI_ROUTER_INTERFACE_ATTR_INGRESS_ACL_LIST;
+    rif_attr_list.list[0].value.oid = acl_table_id2;
+    saistatus = sai_router_interface_api->set_router_interface_attribute(rifid10, 1, rif_attr_list);
+    if (saistatus != SAI_STATUS_SUCCESS) {
+        return saistatus;
+    }
+    
+    // Unbind any ACL from the router interface *rifid10* 
+    rif_attr_list.count = 1;
+    rif_attr_list.list[0].id = SAI_ROUTER_INTERFACE_ATTR_INGRESS_ACL_LIST;
+    rif_attr_list.list[0].value.oid = SAI_NULL_OBJECT_ID;
+    saistatus = sai_router_interface_api->set_router_interface_attribute(rifid10, 1, rif_attr_list);
+    if (saistatus != SAI_STATUS_SUCCESS) {
+        return saistatus;
+    }
+    
+    // Remove the ACL table created earlier *acl_table_id2*
+    saistatus = sai_acl_api->remove_acl_table(acl_table_id2);
+    if (saistatus != SAI_STATUS_SUCCESS) {
+        return saistatus;
+    }
+
+### Example 3 - Create an ACL group 
+This example creates and ACL group with more than one ACL table and bind it to a port, the very same way an ACL table was bound to a port in Example 1. 
+
+    // Create an ingress acl table group 
+    sai_object_id_t acl_grp_id1 = 0ULL;
+    acl_grp_attr[0].id = SAI_ACL_TABLE_GROUP_STAGE;
+    acl_grp_attr[0].value.s32 = SAI_ACL_STAGE_INGRESS;
+
+    acl_attr_list[1].id = SAI_ACL_TABLE_GROUP_ATTR_BIND_POINT;
+    acl_attr_list[1].value.s32 = SAI_ACL_BIND_POINT_PORT;
+
+    acl_grp_attr[2].id = SAI_ACL_TABLE_GROUP_ATTR_PRIORITY;
+    acl_grp_attr[2].value.s32 = 100;
+    saistatus = sai_acl_api->create_acl_table_group(&acl_grp_id1, 2, acl_grp_attr);
+    if (saistatus != SAI_STATUS_SUCCESS) {
+        return saistatus;
+    }
+    
+    // Update the ACL table created in Example 1, to be part of this group
+    acl_attr_list[0].id = SAI_ACL_TABLE_ATTR_GROUP_ID;
+    acl_attr_list[0].value.oid = acl_grp_id1;
+    saistatus = sai_acl_api->set_acl_table_attribute(acl_table_id1, acl_attr_list[0]);
+    if (saistatus != SAI_STATUS_SUCCESS) {
+        return saistatus;
+    }
+    
+    // Create an ACL table *acl_table_id3* , to be part of this group *acl_grp_id1*
+    sai_object_id_t acl_table_id3 = 0ULL;
+    acl_attr_list[0].id = SAI_ACL_TABLE_ATTR_STAGE;
+    acl_attr_list[0].value.s32 = SAI_ACL_STAGE_INGRESS;
+
+    acl_attr_list[1].id = SAI_ACL_TABLE_GROUP_ATTR_BIND_POINT;
```

Comment:
@tushar-ty based on the updated pull request, it should be "SAI_ACL_TABLE_GROUP_ATTR_BIND_POINT".